### ISSUE TITLE
Opacity ignored for minimum contrast rule 1.4.3

### ISSUE THREAD

**Issue by meisterz39** [ThePacielloGroup/ARC-Toolkit#54]
**Describe the bug**

The ARC toolkit can be tricked into ignoring contrast issues so long as the contrast stems from opacity rather than a color value.

**To Reproduce**

1. Take any color with proper contrast, set it to be highly transparent
2. Run ARC against the page

```
body { background-color: #fff }
.color-1 { color: black; } // Passes
.color-2 { color: #ddd; } // Fails
.color-3 { color: black; opacity: 0.2; } // Passes
```

**Expected behavior**

Color contrast tests should account for opacity and correctly fail when a color is too transparent to produce adequate contrast

**Screenshots**

![Uploading image.png…]()

**Version information**

- Browser and version: Chrome Version 121.0.6167.140
- ARC Toolkit version: 5.6.0

**Comment by ferllings** [comment]
Fix already done on rules engine.
Should be fixed in next release

**Comment by poorgeek** [comment]
Closing as this was fixed in ARC Toolkit 5.7.0.
